docs/configuration_files.md

```diff
@@ -32,6 +32,10 @@
     - [Description](#description)
     - [Options](#options)
         - [Example](#example)
+- [Sniping _(Sniper)_](#sniping-_-sniper-_)
```

Comment:
I don't know why, but navigating from the links does not jump to the new sniper section, and the sublinks go to the previous one (for example, new sniper description goes to old sniper description).